pkg/daemon/ceph/client/mirror_test.go

```diff
@@ -107,7 +107,6 @@ func TestImportRBDMirrorBootstrapPeer(t *testing.T) {
 			assert.Equal(t, "bootstrap", args[3])
 			assert.Equal(t, "import", args[4])
 			assert.Equal(t, pool, args[5])
-			assert.Equal(t, "/tmp/rbd-mirror-token-pool-test", args[6])
```

Comment:
creating a file through `ioutil.TempFile` adds some random number at the end of the file, so we cannot use this assertion.